tests/units/classes/test.php

```diff
@@ -1245,5 +1250,51 @@ public function testStaticMethod()
 						->isEqualTo(array($return1, $return2, $return3))
 			;
 		}
+
+		public function testNewMockInstance()
+		{
+			$this
+				->if($test = new emptyTest())
```

Comment:
we use a dummy `emptyTest` class (declared at the top of this file) as the tested class